tvm_utility/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!DOCTYPE xml>
```

Comment:
ROS XML formats don't use a DTD or DOCTYPE, they use a format schema. Please replace this line with

```
<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Reply:
Done.